### ISSUE TITLE
Improve Based On Object view

### ISSUE THREAD

**Issue by BenjaminNSD** [DASISH/qddt-client#255]
When clicking on Based on Object, details, not all details come up in pop up. Please make details complete
![based on](https://cloud.githubusercontent.com/assets/20127077/21391478/2be66ba0-c78c-11e6-9715-1a5951477aff.PNG)

**Comment by liuy97** [comment]
Refs #295, the basedon information will be lost.

**Comment by liuy97** [comment]
This issue depends on #275.

**Comment by liuy97** [comment]
The based-on detail supports question item, response domain, control construct, and concept. The other elements create a new issue please, if the backend supports them.
